### ISSUE TITLE
HUU: Engineering Agenda and Meeting Minutes

### ISSUE THREAD

**Issue by lasryariel** [hackforla/HomeUniteUs#770]
### Overview
This issue tracks the agenda for the Engineering meetings for the HomeUniteUS project

### Minutes Template
```
## Agenda and Minutes 2024-08-29

### Prework
- 

### Roll Call
- [ ] Erik Guntner
- [ ] John Wroge
- [ ] Jose Padilla
- [ ] Paul Espinosa
- [ ] Tyler Thome


#### Recurring items: 
- [ ] Review assignments for each engineer
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Jose](https://github.com/hackforla/HomeUniteUs/issues/assigned/JpadillaCoding)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)

#### Meeting Notes
- .

#### New Tasks
- [ ] 

```

**Comment by lasryariel** [comment]
## Agenda and Minutes 2024-08-29

### Prework
- 

### Roll Call
- [ ] Erik Guntner
- [ ] John Wroge
- [ ] Jose Padilla
- [ ] Paul Espinosa
- [ ] Tyler Thome
- [ ] Ariel Lasry (Engineering PM)

### FYIs/Discussions
- 

#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Jose](https://github.com/hackforla/HomeUniteUs/issues/assigned/JpadillaCoding)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

#### Meeting Notes
- 

#### New Tasks
- [ ]

**Comment by lasryariel** [comment]
## Agenda and Minutes 2024-08-29

### Prework
- 

### Roll Call
- [ ] Erik Guntner
- [ ] John Wroge
- [ ] Jose Padilla
- [ ] Paul Espinosa
- [ ] Tyler Thome
- [ ] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)

### FYIs/Discussions
- meeting time going forward
- Review/Update #771

#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Jose](https://github.com/hackforla/HomeUniteUs/issues/assigned/JpadillaCoding)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

#### Meeting Notes
- 

#### Next Week
Return to these discussions once we are through the current lift.
- [ ] discussion: EventStorming/Event Modeling
- [ ] discussion: picking up Josh's work
- [ ] discussion: John backend implementation work
- [ ] discussion: meeting time going forward
- [ ] discussion: how work is handed off to engineering (sizing/grouping issues) and scoping (potentially create a research issue for Sr engineer to kickoff handoff

**Comment by lasryariel** [comment]
## Agenda and Minutes 2024-09-12

### Prework
- 

### Roll Call
- [ ] Erik Guntner
- [x] John Wroge
- [ ] Jose Padilla
- [x] Paul Espinosa
- [x] Tyler Thome
- [x] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)

### FYIs/Discussions
- meeting time going forward - good with everyone, confirm with Erik
- Review/Update #771

#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Jose](https://github.com/hackforla/HomeUniteUs/issues/assigned/JpadillaCoding)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

#### Meeting Notes
- 

#### Next Week
Return to these discussions once we are through the current lift.
- [ ] discussion: EventStorming/Event Modeling
- [ ] discussion: picking up Josh's work
- [ ] discussion: John backend implementation work
- [ ] discussion: meeting time going forward
- [ ] discussion: how work is handed off to engineering (sizing/grouping issues) and scoping (potentially create a research issue for Sr engineer to kickoff handoff

**Comment by lasryariel** [comment]
## Agenda and Minutes 2024-09-19

### Prework
- 

### Roll Call
- [x] Erik Guntner
- [x] John Wroge
- [ ] Jose Padilla
- [x] Paul Espinosa
- [x] Tyler Thome
- [x] Ariel Lasry (Engineering PM)
- [x] Kiyingi Brian (Engineering PM)

### FYIs/Discussions
- Review/Update #771

#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Jose](https://github.com/hackforla/HomeUniteUs/issues/assigned/JpadillaCoding)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

#### Meeting Notes
- 

#### Next Week
Return to these discussions once we are through the current lift.
- [ ] discussion: EventStorming/Event Modeling
- [ ] discussion: picking up Josh's work
- [ ] discussion: John backend implementation work
- [ ] discussion: meeting time going forward
- [ ] discussion: how work is handed off to engineering (sizing/grouping issues) and scoping (potentially create a research issue for Sr engineer to kickoff handoff

**Comment by lasryariel** [comment]
## Agenda and Minutes 2024-09-19

### Prework
- 

### Roll Call
- [x] Erik Guntner
- [x] John Wroge
- [ ] Jose Padilla
- [x] Paul Espinosa
- [x] Tyler Thome
- [x] Ariel Lasry (Engineering PM)
- [x] Kiyingi Brian (Engineering PM)

### FYIs/Discussions
- Review/Update #771
- Review/Update #790 

#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Jose](https://github.com/hackforla/HomeUniteUs/issues/assigned/JpadillaCoding)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

#### Meeting Notes
- 

#### Next Week
Return to these discussions once we are through the current lift.
- [ ] discussion: EventStorming/Event Modeling
- [ ] discussion: picking up Josh's work
- [ ] discussion: John backend implementation work
- [ ] discussion: meeting time going forward
- [ ] discussion: how work is handed off to engineering (sizing/grouping issues) and scoping (potentially create a research issue for Sr engineer to kickoff handoff

**Comment by lasryariel** [comment]
10/3

Incubator:
- Core compute and network resources deployed
- certificates provisioned
- Remains:
  - Troubleshooting load balancer
  - Cognito still needs to be worked on, module the ops team set up has never been used and needs to be tweaked
  - Check in with Ops 10/9 validate/merge 
  - Have Product/anyone help with testing after merge
  - Leave both instances running for a week or two after validation, then meet with Bonnie to turn off HUU AWS
  - once we are migrated we should create test users to be added to 1Password for anyone to test without login (use plus sign to create variants) 
    - (make sure plus sign is not bugged)
- Risk:
  - Person who originally set up incubator no longer with H4LA, some risk that Ops team makes changes that we will have to adapt to.

Feature Work:
- Erik created prototype for Intake Profile menu

FastAPI migration:
- Close to done on FastAPI migration/controller updates
- would need to connect to google account AND Cognito in order to fully test some changes (invite flow branch)
- Paul expecting to check off most of #789 by Tuesday
- once we are done, convert [this comment](https://github.com/hackforla/HomeUniteUs/issues/789#issuecomment-2365329133) to a wiki
- Google auth endpoints can only be worked on after incubator migration is complete

**Comment by lasryariel** [comment]
## Agenda and Minutes 2024-10-10

### Prework
- 

### Roll Call
- [ ] Erik Guntner
- [x] John Wroge
- [x] Paul Espinosa
- [x] Tyler Thome
- [x] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)

### FYIs/Discussions
- Review/Update #771
- Review/Update #790 

#### Recurring items: 
- [x] Review assignments
  - [x] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [x] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [x] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [x] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [x] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

#### Meeting Notes


#### Next Week
Return to these discussions once we are through the current lift.
- [ ] discussion: EventStorming/Event Modeling
- [ ] discussion: picking up Josh's work
- [ ] discussion: John backend implementation work
- [ ] discussion: meeting time going forward
- [ ] discussion: how work is handed off to engineering (sizing/grouping issues) and scoping (potentially create a research issue for Sr engineer to kickoff handoff

**Comment by lasryariel** [comment]
## Agenda and Minutes 2024-10-17

### Prework
- 

### Roll Call
- [x] Erik Guntner
- [ ] John Wroge
- [ ] Paul Espinosa
- [ ] Tyler Thome
- [x] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)

### FYIs/Discussions
- Review/Update #771
- Review/Update #790 

#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

#### Meeting Notes
- Erik working on making front end components composable/resusable across Guest/Host

Incubator Migration
- everything should be deployed
- create a ticket: need to add API keys to AWS for google authentication, which we will do through a h4la google dev account
  - Might need to add things like TOS/privacy during the set up process, although might not be required for dev environment
- create a ticket: need to generate a build in order for site to be functional, running in to some errors on automated tests that John is working on (Erik to try to help as well)
- need a way for all devs to push builds without having to go to Tyler (github action for anyone to press a button to deploy to dev environment) (would then create a more restricted way to promote from dev to prod) (currently tyler can push because he has access to the devops repo which he won't have in the future)

FastAPI Migration
- Paul submitted PR for github actions so it runs tests on the new API (checks off items from #789)
  - API Error Design can be separated to it's own issue

Next Steps/Order:
- Fix the tests
- merge Paul's PR
- setup google auth api keys
- generate a build for Tyler to manually deploy to qa.homeunite.us
- later we will switch qa to point to dev

#### Next Week
Return to these discussions once we are through the current lift.
- [ ] discussion: EventStorming/Event Modeling
- [ ] discussion: picking up Josh's work
- [ ] discussion: John backend implementation work
- [ ] discussion: meeting time going forward
- [ ] discussion: how work is handed off to engineering (sizing/grouping issues) and scoping (potentially create a research issue for Sr engineer to kickoff handoff

**Comment by lola3736** [comment]
@lasryariel below is the summary meeting notes from Fathom Note Taker:

The following summary and action items were generated by AI
### **_Summary_**
#### **Meeting Purpose**
Review engineering progress and discuss next steps for Home Unite Us project migration to Fast API and incubator readiness.

#### **_Key Takeaways_**
- Most tests are now passing, with only API tests remaining to be fixed
- Google API keys have been set up, but some configuration details need finalization
- The migration to Fast API is nearly complete, with UI and API mostly working
- Sign-up functionality is still broken, presenting a 400 error that needs investigation
- Google sign-in implementation for Fast API is still pending
#### **_Topics_**
- Test Status and Fixes
   - Unit tests and end-to-end tests are now passing
   - John's PR fixes remaining API test failures
   - Paul's PR includes backend test fixes
   - Team to merge PRs and verify all tests pass on main branch
- Google API Configuration
  - Tyler set up API keys tied to the Hack for LA account 
  - Cognito configuration updated via Terraform 
  - Only email and name will be pulled from Google, other user data to be entered manually
  - Documentation of setup process still needed for knowledge transfer
- Migration to Fast API Progress
  - UI and API are mostly working on QA environment (qa.homeunite.us)
  - Sign-up functionality is broken, returning a 400 error
  - Google sign-in button is currently commented out in the UI
  - Need to implement Google sign-in endpoints in Fast API
- Incubator Readiness
  - Project is close to feature parity with the old system
  - Main blocker is resolving the sign-up error and implementing Google sign-in
  - Team needs to create test logins to verify functionality of the rest of the application
#### **_Next Steps_**
- Investigate and fix the 400 error on the sign-up route
- Implement Google sign-in endpoints in Fast API
- Uncomment and test Google sign-in buttons in UI
- Create test logins in Cognito to verify application functionality
- Complete a full parity build and test all features
- Finalize Google API configuration details (e.g., required user data fields)
- Document Google API setup process for knowledge transfer
- Aim to complete migration before the end of the month to avoid additional charges
#### **_Action Item_**
- Merge PR with fixes for Backend tests into main branch - John W		
- Merge John's PR for front-end test fixes into main branch - Paul Espinosa		
- Implement Google sign-in endpoints for Fast API - Tyler Thome		
- Resolve 400 bad request error in sign-up route; implement verbose logging for debugging - Paul Espinosa		
- Uncomment Google sign-in buttons in UI for sign-in and sign-up pages -Erik Guntner

**Comment by lola3736** [comment]
## Agenda and Minutes 2024-10-17

### Prework
- 
### Roll Call
- [x] Erik Guntner
- [x] John Wroge
- [x] Paul Espinosa
- [x] Tyler Thome
- [ ] Lola Sarumi (PM)
- [ ] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)


#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

### Meeting Notes
Meeting Purpose
- Discuss project progress, review action items, and plan next steps for the Home Unity development team.
#### Key Takeaways
 - Significant progress made on incubator migration; most action items completed
 - Google authentication setup nearly complete, with minor documentation needed
 - Team to review and update project board, focusing on new designs and upcoming feature work
 - Preparing for transition to more feature-building and workflow implementation
#### Topics
 - Incubator Migration Progress
 - Paul's PR merged, draft PR for event modeling architecture in progress
 - Google authentication mostly working, with one minor bug remaining
 - Parity builds deployed, marking migration as essentially complete
 - Team at parity with old setup, meeting defined goals for the migration
#### Google Authentication Documentation
 - Documentation not started yet, but code migration nearly complete
 - Agreed to create high-level bullet points initially, covering setup steps:
   1. Set up Cognito
   2. Create Google account and developer account
   3. Set up API and obtain OAuth credentials
 - Tyler to draft initial documentation, to be reviewed by Bonnie later
#### Project Board Review and Updates
 - Team to review and update prioritized backlog, especially older items that may no longer apply due to new designs
 - New issue approval section to be reviewed and updated, particularly infrastructure-related items
 - In-progress items to be evaluated for potential move to completed status
 - Design system ticket to be moved out of engineering section
 - Team to wait for new designs before extensively updating tickets
#### Upcoming Work and Planning
 - Preparing to transition to feature-building and workflow implementation
 - Awaiting wireframes for guest and coordinator workflows to update project board accordingly
 - Team feeling more confident about issue sizing and descriptions
 - Aiming to start actively using the project board for workflow management by January
#### Miscellaneous Updates
 - Meeting time confirmed to remain at 5:30
 - Paul working on event modeling research; team to evaluate its simplicity and onboarding potential before implementation
 - Potential to show a prototype by next June mentioned
#### Next Steps
 - Review and update project board, focusing on new issue approval and in-progress items
 - Wait for new design wireframes before extensively updating prioritized backlog
 - Tyler to draft initial Google authentication setup documentation
 - Continue work on resolving final Google sign-in issue
 - Prepare for transition to feature-building and workflow implementation work
#### Action Items		
 - Draft high-level bullet points for Google auth setup documentation - **_Erik Guntner_**		
 - Debug remaining Google sign-in/sign-up issue - **_Tyler Thome_**		
 - Review prioritized backlog, mark items that may be outdated due to new designs - **_Tyler Thome_**		
 - Review "New Issue Approval" column, update/close infrastructure-related items - **_Erik Guntner_**		
 - Review "In Progress" column, move completed items to "Completed" if PRs are merged - **_John W_**

**Comment by lasryariel** [comment]
## Agenda and Minutes 2024-31-24

### Prework
- 
### Roll Call
- [x] Erik Guntner
- [x] John Wroge
- [x] Paul Espinosa
- [x] Tyler Thome
- [x] Lola Sarumi (PM)
- [ ] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)


#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

### Meeting Notes

**Comment by lasryariel** [comment]
## Agenda and Minutes 2024-11-7

### Prework
- 
### Roll Call
- [ ] Erik Guntner
- [x] John Wroge
- [ ] Paul Espinosa
- [x] Tyler Thome
- [x] Lola Sarumi (PM)
- [x] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)


#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

### Meeting Notes
- Discussed process for engineering engagement during design phase:
  - product and design iterate on wireframes
  - review with stakeholder (bonnie)
  - once core wireframe is approved, present to engineering
  - assign engineer to join breakout for each major feature
  - assigned engineer runs point, responsible for looping in broader engineering team
    - This is _not_ an assignment to write the code, this is just to engage in the design process
  - work towards finalized design with input from engineering, continue to schedule reviews with stakeholder (bonnie) as we iterate

**Comment by lasryariel** [comment]
- recruiting
  - recruit 2 juniors to pair
  - add senior backend role- so now we will have a senior back and front end role open, will see what we can get
  - Tyler to write up description for above roles, Ariel to make postings when descriptions ready
- coordinator updates/engineer to join breakout
  - presented to team
  - Erik to join Coordinator breakout starting next week 
- any other updates
  - pushed a change to fix google sign in, will test today

**Comment by lola3736** [comment]
@lasryariel I added summary meeting minutes

**Comment by lola3736** [comment]
## Agenda and Minutes 2024-11-14
### Roll Call
- [ ] Erik Guntner
- [ ] John Wroge
- [ ] Paul Espinosa
- [ ] Tyler Thome
- [ ] Lola Sarumi (PM)
- [ ] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)

#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

## Summary Meeting Notes/Minutes
 - Meeting Purpose
   - Review project board, discuss workflow processes, and align on development approach for the coordinator dashboard.
### Key Takeaways
 - Project board needs updating to reflect current work status and priorities
 - Workflow process refined: dev leads (Tyler/Erik) to review and approve tickets before moving to prioritized backlog
 - Aim for prototype by June 2025 for stakeholders
 - Coordination wireframe needs approval from Bonnie in order for dev team to proceed with structural work
### Topics
#### Project Board Management
 - Update board to reflect current work status
 - New issue column for drafts; prioritized backlog for approved work
 - One ticket per person in "In Progress" column
 - Close irrelevant tickets with explanatory comments, don't delete
 - Highest priority items should be at the top of prioritized backlog
#### Workflow Process
 - Dev leads (Tyler/Erik) to review and approve all tickets
 - Approved tickets should be moved to prioritized backlog
 - Comment in ticket to indicate review/approval for audit trail
 - Everything currently considered "must have" for MVP
#### Development Timeline and Coordination
 - Unofficial target: prototype for stakeholders by June 2025
 - Discussion needed with Bonnie on dev team involvement in wireframe review
 - Concern about potential rework if dev team reviews too early
 - Tyler suggests parameterizing design elements for easy adjustment
#### Coordinator Dashboard Development
 - Erik to outline work for setting up dashboard structure
 - Create detailed issue for Bonnie's approval before development starts
 - Ensure Tyler reviews and approves before presenting to Bonnie
#### Design and Development Collaboration
 - Balancing conventional UI patterns with Bonnie's desire for unique, user-friendly design
 - Need to avoid reinventing UI widgets unnecessarily
 - Importance of getting core data flow and structure right before detailed design
#### Next Steps
 - Update project board to reflect current work status and priorities
 - Erik to create detailed issue for coordinator dashboard structure setup
 - Schedule meeting with Bonnie to discuss dev team involvement in wireframe review, potentially to discuss during breakout session at the 11/26 "All Teams" meeting.
 - Tyler to review and approve coordinator dashboard structure plan before Bonnie's review
### Action Items		
 - [ ] Review prioritized backlog tickets, ensure complexity & points labels are accurate - Tyler Thome 		
 - [ ] Outline work for coordinator dashboard structure/wireframe - create detailed ticket(s) - Erik Guntner
 - [ ] Present coordinator dashboard structure/wireframe for Bonnie's approval

**Comment by lasryariel** [comment]
## Agenda and Minutes 2024-11-14
### Roll Call
- [ ] Erik Guntner
- [ ] John Wroge
- [ ] Paul Espinosa
- [ ] Tyler Thome
- [ ] Kristen Cardon
- [ ] Lola Sarumi (PM)
- [ ] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)

#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

## Summary Meeting Notes/Minutes

**Comment by lasryariel** [comment]
1/7 Meeting 
- Google sign up and sign in not working
  - sign up: 
    - actually worked with a fresh account! 
      - UI didn't give any confirmation though, just looped back to create account
        - deserialization issue in back end not communicating correctly to front end is preventing user from actually getting logged in even though sign up is successful - Tyler to try this one
  - sign in: Cannot read properties of null (reading 'role') 
    - console error uncaught promise: "Missing refresh token or id token"

- Regular sign up confirms email sent but Ariel not receiving email
- - An error occurred in Confirm Sign up when hitting resend email

- delete old flask codebase

- we don't have to wait to have full parity post migration in order to start on other features (as long as one sign up/sign in option works)
  - But we want to focus on basics of sign up for this 1st month 

To Do
- easier now to give access to see which emails are in db - need devops to give Paul and John access (and maybe ariel), Tyler to request it

- user needs to be in database as well as Cognito, Tyler wants to make this atomic

**Comment by lasryariel** [comment]
## Agenda and Minutes 2024-11-14
### Roll Call
- [x] Erik Guntner
- [x] John Wroge
- [ ] Paul Espinosa
- [ ] Tyler Thome
- [ ] Kristen Cardon
- [ ] Lola Sarumi (PM)
- [x] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)

#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

## Summary Meeting Notes/Minutes
`

**Comment by lasryariel** [comment]
- Potentially worth documenting our issue with sign in/sign up from moving to containerized environment for the benefit of other H4LA teams
- Assisting Erik with issues running HUU app locally
  - Fixes involved removing idp cognito URL from .env file
  - Maybe we should have a troubleshooting tips section in the Readme
- need to build github actions to inject secrets for Postgres connection

**Comment by lasryariel** [comment]
## Agenda and Minutes 2024-01-06
### Roll Call
- [x] Erik Guntner
- [x] John Wroge
- [x] Paul Espinosa
- [x] Tyler Thome
- [ ] Kristen Cardon
- [x] Lola Sarumi (PM)
- [x] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)

#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

## Summary Meeting Notes/Minutes
- Potentially worth documenting our issue with sign in/sign up from moving to containerized environment for the benefit of other H4LA teams
- Assisting Erik with issues running HUU app locally
  - Fixes involved removing idp cognito URL from .env file
  - Maybe we should have a troubleshooting tips section in the Readme
- need to build github actions to inject secrets for Postgres connection

**Comment by lasryariel** [comment]
## Agenda and Minutes 2024-01-30
### Roll Call
- [ ] Erik Guntner
- [x] John Wroge
- [x] Paul Espinosa
- [x] Tyler Thome
- [ ] Kristen Cardon
- [ ] Lola Sarumi (PM)
- [x] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)

#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

## Summary Meeting Notes/Minutes

**Comment by lola3736** [comment]
## Agenda and Minutes 2024-01-23
### Roll Call
- [x] Erik Guntner
- [ ] John Wroge
- [x] Paul Espinosa
- [ ] Tyler Thome
- [x] Lola Sarumi (PM)
- [x] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)

#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

## Summary Meeting Minutes 
- Meeting Purpose
  - Discuss engineering team structure, onboarding new members, and project priorities for HUU.
### Key Takeaways
 - Engineering team to selectively join persona-based breakout sessions, focusing on coordinator utilities development
 - New team members to be introduced to high-level project overview and potentially join breakout sessions for context
 - 2025 goals include launching beta and securing seed funding; immediate focus on prototype development and team expansion
 - Potential new hires (Lucy, Ariel) discussed for frontend and backend roles respectively
### Topics
 - Engineering Team Integration with Persona-Based Sessions
   - Engineers to join persona breakout sessions when there's functional discussion relevant to development
   - Tyler suggested joining when designs are ready to share, before presenting to Bonnie
   - Lola proposed engineers review designs prior to Bonnie's feedback for early technical input
- Onboarding New Team Members
   - Discussed benefits of new engineers sitting in on persona sessions for project context
   - Proposed introductory meetings to cover high-level pipeline and project overview
   - Plan to invite potential new hires (Ariel, others) to next Tuesday's breakout session
- Project Management Updates
  - Bonnie reducing attendance in bi-weekly meetings until substantial progress to present
  - Lola maintaining weekly one-on-ones with Bonnie for updates
  - Coordinator meeting (Tuesdays at 5 or 6 PM) to be cancelled
- 2025 Goals and Immediate Priorities
  - Two main goals for 2025: launch beta and secure seed funding
  - Focus on getting more developers onboard
  - Prototype development as a key milestone
  - Monthly goals to be set aligned with overall objectives
 - Potential New Hires
   - Lucy: Strong frontend experience, potential to assist Erik
   - Ariel: Backend experience in Node.js, could transition to Python if interested
   - Discussed possibility of other HUU projects using Node.js backend
- Ongoing Development Work
  - Some issues still pending in Phase 1, including Google signup functionality
  - Development pace acknowledged as slow during team's "sea legs" period
  - Plan to ramp up productivity in February
### Action Items		
- [ ] Prep high-level project pipeline overview for new team members - **_Erik Guntner_**		
- [ ] Cancel recurring coordinator meeting (previously every other Tue) - **_Tyler Thome_**		
- [ ] Add Ariel + other new team members to HUU Engineering meeting invite - **_Tyler Thome_**		
- [ ] Contact Ariel + other potential new team members re joining next Tue meeting - **_Lola Sarumi_**
### Next Steps
- Tyler to add new team members to meeting invites once emails are obtained
- Lola to reach out to Ariel regarding interest in joining the team
- Revisit and potentially demo the sign-up process (low priority)
 - Continue work on coordinator utilities and address remaining Phase 1 issues
 - Prepare for ramping up development pace in February
 - Set monthly goals aligned with 2025 objectives (beta launch, seed funding)

**Comment by lola3736** [comment]
## Agenda and Minutes 2025-02-06
### Roll Call
- [x] Erik Guntner
- [ ] John Wroge
- [x] Paul Espinosa
- [ ] Tyler Thome
- [x] Lola Sarumi (PM)
- [x] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)

#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

## Summary Meeting Minutes
 - Meeting Purpose
   - Discuss engineering priorities, project workflow, and team updates for the HUU project.
### Key Takeaways
- Need to prioritize and sequence development tasks for intake forms, dashboards, and user flows
- Engineering team to outline structural/backend work needed regardless of specific features
- Tyler working on resolving final bug in account creation and implementing GitHub Actions for faster iteration
- Team considering whether to bring on additional engineering resources
### Topics
 - Project Workflow and Prioritization
 - PM team working on prioritizing next steps for intake form and coordinator dashboard
 - Discussion on sequencing tasks: coordinator invites users, dashboard development, data input, etc.
 - Need to refine user flow diagram to include missing steps (e.g., guest intake form)
 - Lola to draft prioritized workflow and consult with engineering team for feedback
 - Tyler suggested using sequence diagrams with swim lanes to visualize time dimension and task dependencies
- Engineering Infrastructure
   - Account creation setup nearly complete, with one remaining bug to be resolved
   - Tyler focusing on implementing GitHub Actions for deployment to incubator environment
   - Will allow developers without direct AWS access to deploy to Dev/QA environment
   - Expected to increase team velocity and iteration speed
   - Tyler to create an issue documenting this infrastructure work
 - Team Updates and Resources
   - Slow start to the year noted, partly due to holidays and personal matters
   - Discussion about potential new team member (unnamed) who hasn't joined meetings yet
   - Team to evaluate if additional engineering resources are needed
### Action Items
 - [ ] Create GitHub issue for implementing GitHub Actions deployment pipeline to incubator environment. Include explanation of value and velocity increase. - Tyler Thome		
 - [ ] Add Tyler's notes re GitHub Actions deployment to Phantium doc - **_Tyler Thome_**	
 - [ ] Add new team member (Prateek) to this meeting invite - **_Lola Sarumi_**
### Next Steps
- Lola to draft prioritized workflow and share with engineering team for feedback
- Tyler to resolve final bug in account creation setup
- Tyler to create issue documenting GitHub Actions implementation work
- Team to review need for additional engineering resources
- Update meeting notes in chat for next week's reference

**Comment by lola3736** [comment]
## Agenda and Minutes 2025-02-33
### Roll Call
- [x] Erik Guntner
- [ ] John Wroge
- [x] Paul Espinosa
- [ ] Tyler Thome
- [x] Lola Sarumi (PM)
- [x] Ariel Lasry (Engineering PM)
- [ ] Kiyingi Brian (Engineering PM)

#### Recurring items: 
- [ ] Review assignments
  - [ ] [Erik](https://github.com/hackforla/HomeUniteUs/issues/assigned/erikguntner)
  - [ ] [John](https://github.com/hackforla/HomeUniteUs/issues/assigned/johnwroge)
  - [ ] [Paul](https://github.com/hackforla/HomeUniteUs/issues/assigned/paulespinosa)
  - [ ] [Tyler](https://github.com/hackforla/HomeUniteUs/issues/assigned/tylerthome)
  - [ ] [Ariel](https://github.com/hackforla/HomeUniteUs/issues/assigned/lasryariel)

## Summary Meeting Minutes
